pkg/v1/cli/carvelhelpers/package.go

```diff
@@ -13,15 +14,17 @@ import (
 
 // ProcessCarvelPackage processes a carvel package and returns a configuration YAML
 // Downloads package to temporary directory and processes the package by
-// implementing equivalent functionality as the command: `ytt -f <path> | kbld -f -`
-func ProcessCarvelPackage(image string) ([]byte, error) {
-	configDir, err := DownloadImageBundleAndSaveFilesToTempDir(image)
+// implementing equivalent functionality as the command: `ytt -f <path> [-f <values-files>] | kbld -f -`
```

Comment:
general comment is there is enough complexity introduced to this file to warrant some unit tests.

Reply:
Added unit tests